odl/contrib/datasets/ct/examples/mayo_reconstruct.py

```diff
@@ -32,16 +41,27 @@
 td_window = odl.tomo.tam_danielson_window(ray_trafo, n_pi=3)
 
 # Calculate FBP reconstruction
-fbp_result = fbp(td_window * proj_data)
+start = perf_counter()
+fbp_result = fbp(td_window * flat_proj_data)
+stop = perf_counter()
+print('FBP done after {:.3f} seconds'.format(stop-start))
+
+fbp_result_HU = (fbp_result-0.0192)/0.0192*1000
+
+# Save reconstruction in Numpy format
+fbp_filename = proj_folder+'/fbp_result.npy'
+print("Saving reconstruction data in {:s}".format(fbp_filename))
+np.save(fbp_filename, fbp_result_HU)
```

Comment:
Better not save data to disk in an example. And if so, not in the original dataset directory.
For an example, I think it's better to just display it. I like the HU-converted reconstruction.